esp-hal/MIGRATING-0.22.md

```diff
@@ -158,3 +158,26 @@ is enabled. To retrieve the address and size of the initialized external memory,
 ```
 
 The usage of `esp_alloc::psram_allocator!` remains unchanged.
+
+
+### embedded-hal 0.2.* is not supported anymore.
```

Comment:
Might be worth to leave a link to https://github.com/rust-embedded/embedded-hal/blob/master/docs/migrating-from-0.2-to-1.0.md?

Reply:
Oh, thank you!
I didn't even realize it existed haha 😄